app/grants/models.py

```diff
@@ -59,6 +59,24 @@ def __str__(self):
         """Return the string representation of a Grant."""
         return f"id: {self.pk}, status: {self.status}, title: {self.title}, description: {self.description}"
 
+
+class Milestone(SuperModel):
+    """Define the structure of a Grant Milestone"""
```

Comment:
Can you throw a new line between the docstring and the attributes here?

Reply:
:+1: